tests/AdaptiveExpressions.Tests/ExpressionParserTests.cs

```diff
@@ -635,7 +635,7 @@ public class ExpressionParserTests
             Test("formatNumber(12.123, 2)", "12.12"),
             Test("formatNumber(1.551, 2)", "1.55"),
             Test("formatNumber(12.123, 4)", "12.1230"),
-            Test("formatNumber(12000.3, 4, 'fr-fr')", "12\x00a0000,3000"),
+            Test("formatNumber(12000.3, 4, 'fr-fr')", "12\x202F000,3000"),
             #endregion
```

Comment:
There is something odd here.  Your fix fails on our build machines when they run the Mac/Linux profile.  You can see this in the build logs from your PR page. The question is why your results are different.  I'm not that familiar with the Mac, but is there the equivalent of Windows language packs?  Is it possible that it depends on whether or not you have that installed?
 X Evaluate (formatNumber(12000.3, 4, 'fr-fr'),12â€¯000,3000,) [27ms]
2020-05-16T20:55:28.5352450Z   Error Message:
2020-05-16T20:55:28.5353480Z    Assert.AreEqual failed. Expected:<12â€¯000,3000>. Actual:<12Â 000,3000>. 
2020-05-16T20:55:28.5354020Z   Stack Trace:
2020-05-16T20:55:28.5354700Z      at AdaptiveExpressions.Tests.ExpressionParserTests.AssertObjectEquals(Object expected, Object actual) in /Users/runner/runners/2.168.2/work/1/s/tests/AdaptiveExpressions.Tests/ExpressionParserTests.cs:line 1181
2020-05-16T20:55:28.5355800Z    at AdaptiveExpressions.Tests.ExpressionParserTests.Evaluate(String input, Object expected, HashSet`1 expectedRefs) in /Users/runner/runners/2.168.2/work/1/s/tests/AdaptiveExpressions.Tests/ExpressionParserTests.cs:line 964